internal/iam/command_rbac_role_binding_list.go

```diff
@@ -88,28 +88,28 @@ func (c *roleBindingCommand) newListCommand() *cobra.Command {
 		)
 	}
 
-	cmd.Flags().String("principal", "", "Principal whose role bindings should be listed.")
-	cmd.Flags().Bool("current-user", false, "Show role bindings belonging to the current user.")
-	cmd.Flags().String("role", "", "List role bindings under a specific role given to a principal. Or if no principal is specified, list principals with the role.")
+	cmd.Flags().String("principal", "", "Principal ID, which limits role bindings to this principal. If unspecified, list all principals and role bindings.")
```

Comment:
The principal is not exactly an ID. It could look like `User:u-12345` which _contains_ an ID.

Reply:
Updated -- that was pending a response in the #cli channel.